spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -30,6 +30,12 @@ private SparkReadOptions() {
   // Snapshot ID of the table snapshot to read
   public static final String SNAPSHOT_ID = "snapshot-id";
 
+  // Start snapshot ID used in incremental scans (exclusive)
+  public static final String START_SNAPSHOT_ID = "start-snapshot-id";
+
+  // End snapshot ID used in incremental scans (inclusive)
+  public static final String END_SNAPSHOT_ID = "end-snapshot-id";
```

Comment:
these 2 are missing corresponding methods in the `SparkReadConf`, is that intentional?

Reply:
These 2 are purely read configs. There is no table or session config.
For now, I just replaced hard-coded values with constants.